src/transformers/models/xlnet/modeling_xlnet.py

```diff
@@ -1443,7 +1443,7 @@ def forward(
             output_attentions=output_attentions,
             output_hidden_states=output_hidden_states,
             return_dict=return_dict,
-            **kwargs,
+            # **kwargs,
```

Comment:
Why are those commented out? This will break backward compatibility with the deprecation cycle inside XLNet for `use_cache`.

Reply:
This makes tracing fail, I readded the kwargs, and will add support for XLNet once this is solved.

Reply:
This makes tracing fails, I re-added the kwargs and will enable tracing for XLNet once this is solved.